snippets/oadp-incremental-backup-support-snippet.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+
+
+:_mod-docs-content-type: SNIPPET
+
+[NOTE]
+====
+{oadp-short} supports incremental backups of `block` and `Filesystem` persistent volumes for both containerized, and {VirtProductName} workloads. 
+
+For `block` volumes, you can use the Container Storage Interface (CSI) Data Mover to perform incremental backups. File system backup is not supported for `block` volumes.
+
+For `Filesystem` volumes, you can use {oadp-short} file system backup, Container Storage Interface (CSI), or CSI Data Mover for incremental backups. 
```

Comment:
I think we should remove the filesystem backup word from here as it's not supported.  

```suggestion
For `Filesystem` volumes, you can use {oadp-short} Container Storage Interface (CSI), or CSI Data Mover for incremental backups. 
```